tests/blockchain/deposit_strategy/test_base_deposit_strategy.py

```diff
@@ -23,16 +23,21 @@ def test_deposited_keys_amount(base_deposit_strategy):
 
 @pytest.mark.unit
 def test_deposited_keys_amount_mellow(mellow_deposit_strategy):
```

Comment:
Rename file to smth like `test_deposit_strategy_keys_amount`

Reply:
Done